lib/guardian/topic_guardian.rb

```diff
@@ -210,6 +210,12 @@ def can_perform_action_available_to_group_moderators?(topic)
   end
   alias :can_archive_topic? :can_perform_action_available_to_group_moderators?
   alias :can_close_topic? :can_perform_action_available_to_group_moderators?
+  alias :can_split_merge_topic? :can_perform_action_available_to_group_moderators?
   alias :can_edit_staff_notes? :can_perform_action_available_to_group_moderators?
 
+  def can_move_posts?(topic)
```

Comment:
I think we need to check that they have access to the source AND destination topic here right?